.github/component_owners.yml

```diff
@@ -23,14 +23,17 @@ components:
   src/OpenTelemetry.Extensions.AWS/:
     - srprash
     - atshaw43
```

Comment:
@ppittle, @srprash - side topic.

@atshaw43 is still not OTel member. He never accepted invitation to OpenTelemetry org. Could you please help with this or if he is no longer responsible for this package, he could self remove.